WildernessLabs/Meadow.CLI
# Support previous HCOM protocol version

To test this
`meadow flash os -v 0.9.4.0`

then use normal commands and it should auto-downgrade to the previous protocol version, without throwing up the repeat error message.
